CHANGELOG.adoc

```diff
@@ -36,6 +36,7 @@ When end-users click on _New Object_ on a semantic element, and select a `ViewUs
 - https://github.com/eclipse-syson/syson/issues/1210[#1210] [general-view] Add the `parameters` compartment on `ActionDefinition` in the _General View_ diagram.
 - https://github.com/eclipse-syson/syson/issues/1267[#1267] [general-view] Improve the New Perform tool selection dialog to display existing actions in a tree.
 - https://github.com/eclipse-syson/syson/issues/1257[#1257] [general-view] Add the `perform actions` compartment on `Part` in the _General View_ diagram.
+- https://github.com/eclipse-syson/syson/issues/1261[#1261] [import] Improve "New Object from Text" action in order to be able to resolve names against existing elements.
```

Comment:
"New Object from Text" => _New objects from text_ (no double quotes, but italic and the same menu item name than in the UI)

Reply:
Done